CMakeLists.txt

```diff
@@ -14,6 +14,10 @@
 
 cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
 
+if(NOT DEFINED BUILD_TESTING)
+  set(BUILD_TESTING OFF)  # Set this to OFF by default
+endif()
```

Comment:
Suggest: Move this below `project()`. Will also want to `include(CTest)`.

Reply:
Based off our conversation in slack, we'll leave off the `include(CTest)` for now since it is currently causing the tests to be built unconditionally.